docs/source/tts/configs.rst

```diff
@@ -0,0 +1,263 @@
+
+.. code-block:: yaml
+
+  model:
+    preprocessor:
+      _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+      features: 80
+      lowfreq: 0
+      highfreq: null
+      n_fft: 2048
+      n_window_size: 2048
+      window_size: false
+      n_window_stride: 512
+      window_stride: false
+      pad_to: 1
+      pad_value: 0
+      sample_rate: 44100
+      window: hann
+      normalize: null
+      preemph: null
+      dither: 0.0
+      frame_splicing: 1
+      log: true
+      log_zero_guard_type: add
+      log_zero_guard_value: 1e-05
+      mag_power: 1.0
+
+Text Normalizer Configuration
+------------------------------
+Text normalization (TN) converts text from written form into its verbalized form, and it is an essential preprocessing step before text-to-speech Synthesis. TN ensures that TTS can handle all input texts without skipping unknown symbols. For example, "$123" is converted to "one hundred and twenty-three dollars". Currently, NeMo supports text normalizers for English, Russian, German, Spanish, and Chinese. Refer to the previous Section :doc:`../nlp/text_normalization/intro` for more details. Below shows an example of specifying text normalizer for English.
+
+.. code-block:: yaml
+
+  model:
+    text_normalizer:
+      _target_: nemo_text_processing.text_normalization.normalize.Normalizer
+      lang: en
+      input_case: cased
+      whitelist: "nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv"
+
+    text_normalizer_call_kwargs:
+      verbose: false
+      punct_pre_process: true
+      punct_post_process: true
+
+Tokenizer Configuration
+------------------------
+Tokenization converts input text string to a list of integer tokens. It may pad leading and/or trailing whitespaces to a string. NeMo tokenizer supports grapheme-only inputs, phoneme-only inputs, or a mixer of grapheme and phoneme inputs to disambiguate pronunciations of heteronyms for English, German, and Spanish. It also utilizes a grapheme-to-phoneme (G2P) tool to transliterate out-of-vocabulary (OOV) words. Please refer to the Section :doc:`../text_processing/g2p/g2p` and `TTS tokenizer collection <https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py>`_ for more details. The following example sets up a ``EnglishPhonemesTokenizer`` with a mixer of grapheme and phoneme inputs where each word shown in the heteronym list is transliterated into graphemes or phonemes by a 50% chance.
```

Comment:
Re: `It also utilizes a grapheme-to-phoneme (G2P) tool to transliterate out-of-vocabulary (OOV) words.` If this refers to the models from Section :doc:`../text_processing/g2p/g2p`, we haven't integrated them into the TTS tokenizers pipeline yet.

Reply:
yes. I know this status because I was aware of the ByT5 G2P model in planning. I can add something like `(the integration to TTS tokenizers pipeline is upcoming soon)`.